lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -791,6 +791,52 @@ struct RvalueExprVisitor {
     return visitAssignmentPattern(expr, *count);
   }
 
+  Value visit(const slang::ast::StreamingConcatenationExpression &expr) {
+    SmallVector<Value> operands;
+    for (auto stream : expr.streams()) {
+      auto value = context.convertRvalueExpression(*stream.operand);
+      if (!value)
```

Comment:
I noticed that each `stream` can also have a `withExpr` and `constantWithWidth`. Could you add error messages if either of these are set, and tell the user that we don't support those cases? It's important that we only succeed if we were able to process everything the user has type, and not silently ignore parts of the input.